torchvision/datasets/video_utils.py

```diff
@@ -49,9 +67,11 @@ class VideoClips(object):
             on the resampled video
     """
     def __init__(self, video_paths, clip_length_in_frames=16, frames_between_clips=1,
-                 frame_rate=None, _precomputed_metadata=None, num_workers=1):
+                 frame_rate=None, _precomputed_metadata=None, num_workers=1,
+                 _backend="pyav"):
```

Comment:
Instead of passing a `_backend` argument, what about just using the `get_video_backend` to get this? Would this be enough for your use-case?

Reply:
I'm sending a commit changing `backend` to `_backend` in the self, so no need to do anything here.